versioned_docs/version-v1.3.0/userguide/scheduling/override-policy.md

```diff
@@ -91,29 +91,6 @@ spec:
 ```
 It means override rules above will only be applied to those resources propagated to clusters which has the `spec.region` field with values in [cn-north-1].
 
-### fieldSelector
-
-#### Examples
-```yaml
-apiVersion: policy.karmada.io/v1alpha1
-kind: OverridePolicy
-metadata:
-  name: example
-spec:
-  ...
-  overrideRules:
-    - targetCluster:
-        fieldSelector:
-          matchExpressions:
-            - key: region
-              operator: In
-              values:
-                - cn-north-1
-      overriders:
-      ...
-```
-It means override rules above will only be applied to those resources propagated to clusters which has the `spec.region` field with values in [cn-north-1].
-
```

Comment:
Nice finding!
The version-1.3.0 contains the redundant thing, thanks for doing this.
But could you please also remove it from the zh one:
https://github.com/karmada-io/website/blob/main/i18n/zh/docusaurus-plugin-content-docs/version-v1.3.0/userguide/scheduling/override-policy.md#fieldselector-1

Reply:
update